LICENSE

```diff
@@ -1,6 +1,7 @@
     MIT License
 
-    Copyright (c) Microsoft Corporation. All rights reserved.
+    Copyright (c) 2018-present Microsoft Corporation. All rights reserved.
```

Comment:
This follows the convention of PyTorch, see https://github.com/pytorch/pytorch/blob/main/LICENSE

Reply:
The feedback from Justin is that it is ok, and that we can remove the "All right reserved"